kedro_azureml/config.py

```diff
@@ -52,6 +80,11 @@ class KedroAzureRunnerConfig(BaseModel):
     account_name: "{storage_account_name}"
     # Name of the storage container
     container: "{storage_container}"
+  resources:
+    __default__:
```

Comment:
Three things here:
1. Let's remove `azure.cluster_name` for the purpose of `resources.__default__` to have this setting only in one place.
1. I would rename `resources` to `compute`, as `compute` is used in Azure SDK, so it will be much clearer for the users.
2. I would not generate `chunky` as a template here, maybe let's stick to something self-commenting like
```yaml
your_node_tag:
    cluster_name: high-cpu-cluster
```
\+ please provide description of the `compute` node in form of a `# comment` like in other nodes in this template.